Automattic/jetpack
# Update Dashboard Assistant style

#### Changes proposed in this Pull Request:

This PR updates the styles used for the Assistant

#### Jetpack product discussion


Context: pbNhbs-1xX-p2

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






_Note: if you run this PR locally, make sure to build the plugin as well as the `js-package/base-style` package._

- Open the browser console
- After connecting the Jetpack plugin to your account, go through the Assistant flow and make sure it matches the screencast below (recorded with a free plan). As you go through each step, make sure the screen looks good on mobile.
- Make sure no error was logged in the console

https://user-images.githubusercontent.com/1620183/156615947-02e584ca-c8fd-4283-bc91-6bb3fa1547fa.mov

#### Other screens references

- Assistant banner on the Dashboard page
<img width="1433" alt="Screen Shot 2022-03-03 at 11 56 46 AM" src="https://user-images.githubusercontent.com/1620183/156616068-48a280f0-b6d2-4636-9cb8-064c93152060.png">

- Summary with activated features
<img width="1008" alt="Screen Shot 2022-03-03 at 11 49 11 AM" src="https://user-images.githubusercontent.com/1620183/156616563-32bd9f75-f44c-4d9e-90f2-4359d930cd36.png">

- Generic upsell card in summary
<img width="470" alt="Screen Shot 2022-03-03 at 11 50 39 AM" src="https://user-images.githubusercontent.com/1620183/156616402-53e0e433-beaf-4ee5-b52c-de8a036a92a7.png">

- _Manage Security_ card in summary
<img width="473" alt="Screen Shot 2022-03-03 at 11 51 28 AM" src="https://user-images.githubusercontent.com/1620183/156616443-69846d90-ecc4-47e5-b59a-2bc62a1c3323.png">

- _One click restores_ card in summary
<img width="471" alt="Screen Shot 2022-03-03 at 11 51 05 AM" src="https://user-images.githubusercontent.com/1620183/156616459-f2e81c67-6391-409e-9833-666552bdecd1.png">

- _Mobile app_ card in summary
<img width="473" alt="Screen Shot 2022-03-03 at 11 52 16 AM" src="https://user-images.githubusercontent.com/1620183/156616489-f91fab89-d7e6-4282-9c06-59d65370f739.png">
